resources/views/layouts/sidebars/_quick-links.blade.php

```diff
@@ -1,34 +1,6 @@
-@foreach ($currentCampaign->menuLinks()->with(['target'])->ordered()->get() as $menuLink)
+@foreach ($links as $menuLink)
     <?php /** @var \App\Models\MenuLink $menuLink */ ?>
-    @if (isset($childName) && $menuLink->parent == $childName || !isset($childName) && $menuLink->parent == $name || $menuLink->parent == $name && $menuLink->parent == $childName )
-        @if ($menuLink->dashboard && $currentCampaign->boosted() && $menuLink->isValidDashboard())
-            <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }} {{ $menuLink->customClass($currentCampaign) }}">
-                <a href="{{ $menuLink->getRoute() }}">
-                    <i class="{{ $menuLink->icon() }}"></i>
-                   {{ $menuLink->name }}
-                </a>
-            </li>
-        @elseif ($menuLink->target)
-            <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }} {{ $menuLink->customClass($currentCampaign) }}">
-                <a href="{{ $menuLink->getRoute() }}">
-                    <i class="{{ $menuLink->icon() }}"></i>
-                    {{ $menuLink->name }}
-                </a>
-            </li>
-        @elseif ($menuLink->type)
-            <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }} {{ $menuLink->customClass($currentCampaign) }}">
-                <a href="{{ $menuLink->getRoute() }}">
-                    <i class="{{ $menuLink->icon() }}"></i>
-                    {{ $menuLink->name }}
-                </a>
-                </li>
-        @elseif ($menuLink->isRandom())
-            <li class="subsection sidebar-quick-link sidebar-quick-link-{{ $menuLink->position }} {{ $sidebar->activeMenuLink($menuLink) }} {{ $menuLink->customClass($currentCampaign) }}">
-                <a href="{{ route('menu_links.random', $menuLink) }}">
-                    <i class="{{ $menuLink->icon() }}"></i>
-                    {{ $menuLink->name }}
-                </a>
-            </li>
-        @endif
+    @if ((isset($childName) && $menuLink->parent == $childName) || (!isset($childName) && $menuLink->parent == $name) || ($menuLink->parent == $name && $menuLink->parent == $childName) )
```

Comment:
I'm confused between the first and second check. I don't see where `$name` is declared. Also if there is a parent but the campaign isn't boosted, the menu link will never be displayed (I think)

Reply:
`$name` is declared on line 45 of `sidebars/app.blade.php`  
`@foreach ($sidebar->campaign($currentCampaign)->layout() as $name => $element)`, the checks are implemented like that to avoid showing the link multiple times when its under a section like world. On line 15 of `quick-links.blade` it checks to render links with parent and no boost.